plugins/processors/execd/examples/count.go

```diff
@@ -0,0 +1,24 @@
+package main
```

Comment:
The only reason you'd write a execd processor is to modify metrics.  This example doesn't do that so I'm concerned that it will mislead beginners.

Reply:
removed